v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/SourceDbToSpannerOptions.java

```diff
@@ -292,4 +292,16 @@ public interface SourceDbToSpannerOptions extends CommonTemplateOptions {
   Options.RpcPriority getSpannerPriority();
 
   void setSpannerPriority(Options.RpcPriority value);
+
+  @TemplateParameter.Text(
+      order = 24,
+      optional = true,
+      description =
+          "Hint for number of uniformization stages. Currently Applicable only for jdc based sources like MySql or PG. Leave 0 or default to disable uniformization. Set to -1 for a log(numPartition) number of stages.",
+      helpText =
+          "Hint for number of uniformization stages. Currently Applicable only for jdc based sources like MySql or PG. Leave 0 or default to disable uniformization. Set to -1 for a log(numPartition) number of stages.")
```

Comment:
jdbc* based source

Reply:
Can we get some heuristic on how setting this would effect the pipeline for end users?

The help text could talk about when should I be enabling uniformization, when -1 and when something > 0 and what value.

Reply:
Thanks. Added more details in the help text.